app/lib/rocketchat.js

```diff
@@ -169,6 +169,17 @@ const RocketChat = {
 			message: I18n.t('Not_RC_Server', { contact: I18n.t('Contact_your_server_admin') })
 		};
 	},
+	async getServerTimeSync(server) {
+		try {
+			const response = await RNFetchBlob.fetch('GET', `${server}/_timesync`);
```

Comment:
I use the same way to fetch to server as: https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/3667/files#diff-88ed18506a065f3ea03357ad43d30e19ac28e6725abb7ba2477eb9c535fa781bR131

Is there another best way to do this?

Reply:
This way is fine